nautobot/core/testing/views.py

```diff
@@ -655,8 +656,8 @@ def test_list_objects_filtered(self):
                 # TODO: https://github.com/nautobot/nautobot/issues/2580
                 #       This is fragile as we move toward more autogenerated test fixtures,
                 #       as "instance.name" may appear coincidentally in other page text if we're not careful.
-                self.assertIn(instance1.name, content, msg=content)
-                self.assertNotIn(instance2.name, content, msg=content)
+                self.assertRegex(content, r">\s*" + re.escape(instance1.name) + r"\s*<", msg=content)
+                self.assertNotRegex(content, r">\s*" + re.escape(instance2.name) + r"\s*<", msg=content)
```

Comment:
Do you think this change is sufficient to resolve #2580 for now (and permit removing the above TODO)?

Reply:
It's probably good enough. Probably needs to be backported to 1.5 though?

Reply:
Wouldn't hurt